src/main/environment/common_local.properties

```diff
@@ -14,6 +14,9 @@ carestreamOrderCreateURL =http://10.208.122.38:8080/commonapi-v1.0/carestream/cr
 #registrationUrl =http://localhost:8080/beneficiary/create
 registrationUrl =http://10.208.122.38:8080/commonapi-v1.0/beneficiary/create
 
+#syncSearchByLocation =http://localhost:8082/id/searchByDistrictId
+syncSearchByLocation =http://10.208.122.32:8082/id/searchByDistrictId
```

Comment:
check this once, it is wrong url i think

Reply:
what is the right url ? @Shrey475  do you know the right url please ?

Reply:
You may need to mention the service name in the url. Like for 'registrationUrl' above. If calling 'Identity' service, you can check for any pre-existing URLs that call the Identity service and use it that way.

Reply:
Change done. You can review again please @roopesh-beehyv @Shrey475